### PR TITLE
Docs migrates3 dh

### PR DESCRIPTION
**Pull request by dhirsch1**
Migration of the updated Amazon S3 Connector doc to the Mulesoft site.

### CODE REVIEW

File: modules/ROOT/pages/amazon/amazon-s3-connector-config-topics.adoc [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+== Create Object Operation
+
+In the `Create Object` operation, set the `Content Length` to a value greater than zero. If the `Content Length` is set to zero (0), the `Create Object` operation creates a zero-byte object.
+
+== To Use an AWS KMS Master Key
```
Comment (george-hoffer) [inline modules/ROOT/pages/amazon/amazon-s3-connector-config-topics.adoc]: Would you be ok dropping "To" from this title and the next?

File: modules/ROOT/pages/amazon/amazon-s3-connector-design-center.adoc [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+To configure a trigger:
+
+. In Design Center, click *Create*.
+. Click *Create new application*.
+. Specify a value for *Project name*, and click *Create*.
+. Exit from *Let's get started* by clicking *Go straight to canvas*.
+. Click on the Trigger card.
+. If you are using the connector as an input source trigger, search for the connector and select an operation. Otherwise, search for and select *HTTP* or *Scheduler*.
+. Complete the fields with the values you want to configure for your trigger.
```
Comment (george-hoffer) [inline modules/ROOT/pages/amazon/amazon-s3-connector-design-center.adoc]: Please provide more details about the fields. No customer benefits from a generic statement like "complete the fields" -- they know that, but they need more info about what info they need to complete the fields. There are most likely similar fields between each operation. Just cover those. This is especially true for this connector with only 2 trigger operations -- if we can't cover 2 operations, what's the point of documenting the use of DC?

File: modules/ROOT/pages/amazon/amazon-s3-connector-design-center.adoc [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
++
+You can use the following Amazon S3 operations as triggers:
++
+* *On Deleted Object* - Initiates access to your app when an Amazon S3 object is deleted.
+* *On New Object* - Initiates access to your app when an Amazon S3 object is created.
++
+Alternatively, you can specify the trigger as an HTTP Listener as input to your application from a browser or an application like Postman or curl, or you can use Scheduler to provide input to your application at a specific time.
```
Comment (george-hoffer) [inline modules/ROOT/pages/amazon/amazon-s3-connector-design-center.adoc]: This sentence is missing info. You state that you can specify a trigger as an HTTP Listener, but then the next phrase is unclear if you're describing how the trigger or the Listener is accessed.

File: modules/ROOT/pages/amazon/amazon-s3-connector-design-center.adoc [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+== Configure the Target Component
+
+. Click “+” next to the trigger card.
+. In *Select a component*, search for and select the connector name.
+. Select an operation for the connector.
+. Enter the required values in the *General* tab.
```
Comment (george-hoffer) [inline modules/ROOT/pages/amazon/amazon-s3-connector-design-center.adoc:41]: Again, please don't make this doc so generic that no actual fields are described. If you cover the info in later sections, provide links to those sections.

File: modules/ROOT/pages/amazon/amazon-s3-connector-design-center.adoc [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
++
+. If needed, enter values for other tabs.
+. Specify access information to the connector resource, as described below.
+. Click *Test* to test the connection.
++
+Validating a connection with Test Connection requires that you have permission in AWS IAM to the action s3:ListAllMyBuckets. If you don’t have this permission, the test fails. However, you can still use the connector and the operations to which you have access.
++
+Access to operations on Amazon S3 is further controlled through policies. It is not always possible to validate your credentials before the exact operation for which you have access completes. This can vary based on the bucket name and other parameters. For example, the test connection can fail if your credentials have a restricted policy.
+
+
```
Comment (george-hoffer) [inline modules/ROOT/pages/amazon/amazon-s3-connector-design-center.adoc:55]: Need a transition here. Users don't see the Connection Type until their 3rd screen: Trigger Card > Set up > Connection Type. Just launching into discussing Connection Types is too hard to follow. Remember too that Design Center users are considered starting users.

File: modules/ROOT/pages/amazon/amazon-s3-connector-design-center.adoc [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+=== Basic Connection Type
+
+Complete the following fields for the *Basic* connection type:
```
Comment (george-hoffer) [inline modules/ROOT/pages/amazon/amazon-s3-connector-design-center.adoc:58]: Say why one uses the Basic connection type.

File: modules/ROOT/pages/amazon/amazon-s3-connector-design-center.adoc [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+|Parameter |Description
+|Configuration Name |Name used to reference the connector instance.
+|Access Key |Alphanumeric text string that uniquely identifies the user who owns the AWS account.
+|Secret Key |Key that plays the role of a password.
+|Try Default AWS Credentials Provider Chain |(Optional) Indicates whether to use temporary credentials.
+|Region Endpoint |RL entry point for a web service.
+|===
+
+=== Role Connection Type
+
```
Comment (george-hoffer) [inline modules/ROOT/pages/amazon/amazon-s3-connector-design-center.adoc:71]: Say why to use this connection type.

File: modules/ROOT/pages/amazon/amazon-s3-connector-design-center.adoc [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+=== Basic Connection Type
+
+Complete the following fields for the *Basic* connection type:
+
+[%header%autowidth.spread]
+|===
+|Parameter |Description
+|Configuration Name |Name used to reference the connector instance.
+|Access Key |Alphanumeric text string that uniquely identifies the user who owns the AWS account.
```
Comment (george-hoffer) [inline modules/ROOT/pages/amazon/amazon-s3-connector-design-center.adoc:64]: Say where users can get the access key

File: modules/ROOT/pages/amazon/amazon-s3-connector-design-center.adoc [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+|Secret Key |Key that plays the role of a password.
```
Comment (george-hoffer) [inline modules/ROOT/pages/amazon/amazon-s3-connector-design-center.adoc:65]: Say where users can locate the secret key.

File: modules/ROOT/pages/amazon/amazon-s3-connector-design-center.adoc [hunk 10]
```diff
@@ -0,0 +1,92 @@
...
+|Try Default AWS Credentials Provider Chain |(Optional) Indicates whether to use temporary credentials.
+|Region Endpoint |RL entry point for a web service.
+|===
+
+=== Role Connection Type
+
+Complete the following fields for the *Role* connection type:
+
```
Comment (george-hoffer) [inline modules/ROOT/pages/amazon/amazon-s3-connector-design-center.adoc:73]: This table is too vague about how a user gets the info. The UI provides more info than this doc.

File: modules/ROOT/pages/amazon/amazon-s3-connector-examples.adoc [hunk 11]
```diff
@@ -0,0 +1,155 @@
...
+[[store-and-retrieve]]
+== Use Case: Store and Retrieve an Image from a URL to Amazon S3
+
+Store an image from a URL in Amazon S3, and retrieve and display the image.
+For this example to work in Anypoint Studio, you must provide Amazon Web Services credentials. You can either replace the variables with their values in the code, or you can provide the values for each variable in the `src/main/resources/mule-artifact.properties` file.
+
+image::amazon/amazon-s3-02.png[Studio 7 Visual Studio Icon Flow]
+
+. Create a new Mule project in Studio.
+. Drag an HTTP connector into the canvas, and select it to open the properties editor.
```
Comment (george-hoffer) [inline modules/ROOT/pages/amazon/amazon-s3-connector-examples.adoc]: This is a Mule 3 statement -- in Studio 7, you drag a connector operation to the canvas.

File: modules/ROOT/pages/amazon/amazon-s3-connector-examples.adoc [hunk 12]
```diff
@@ -0,0 +1,155 @@
...
++
+image::amazon/amazon-s3-04.png[HTTP Connector - General Tab]
++
+[%header%autowidth.spread]
+|===
+|Field |Value
+|Display Name |Enter a name for the connector instance. In this example, the
+HTTP connector is named Get MuleSoft logo.
+|Configuration |Click the green plus symbol and create a new configuration with the *Host* as `developer.mulesoft.com`, *Protocol+ as HTTPS, and *Port* as 80.
+|URL or Path |Set to ``/sites/all/themes/muletheme/images/mulesoft_dev_logo_v2.svg`.
```
Comment (george-hoffer) [inline modules/ROOT/pages/amazon/amazon-s3-connector-examples.adoc]: Typo: Shouldn't have 2 tick marks -- should be 1.

File: modules/ROOT/pages/amazon/amazon-s3-connector-examples.adoc [hunk 13]
```diff
@@ -0,0 +1,155 @@
...
+. Add an HTTP Connector to request the MuleSoft logo from MuleSoft.
++
+image::amazon/amazon-s3-04.png[HTTP Connector - General Tab]
++
+[%header%autowidth.spread]
+|===
+|Field |Value
+|Display Name |Enter a name for the connector instance. In this example, the
+HTTP connector is named Get MuleSoft logo.
+|Configuration |Click the green plus symbol and create a new configuration with the *Host* as `developer.mulesoft.com`, *Protocol+ as HTTPS, and *Port* as 80.
```
Comment (george-hoffer) [inline modules/ROOT/pages/amazon/amazon-s3-connector-examples.adoc]: Typo: The plus sign should be an asterisk to bold Protocol.

File: modules/ROOT/pages/amazon/amazon-s3-connector-studio.adoc [hunk 14]
```diff
@@ -0,0 +1,78 @@
...
+
+To configure a connector in Anypoint Studio:
+
+. Install the connector.
+. Configure the connector.
+. Configure an input source for the connector.
+
+== Install Amazon S3
+
+. In Anypoint Studio, click the Exchange icon in the Studio task bar.
```
Comment (george-hoffer) [inline modules/ROOT/pages/amazon/amazon-s3-connector-studio.adoc]: Please use the latest steps from the template:

. In Studio, create a Mule Project.
. Click the Exchange *(X)* icon in the Studio task bar.
. In Exchange, click *Login* and supply your Anypoint Platform username and password.
. In Exchange, select *All assets* and search for "<connector>".
. Select <connector>, click *Add to project*.
. Follow the prompts to install the connector.

File: modules/ROOT/pages/amazon/amazon-s3-connector-studio.adoc [hunk 15]
```diff
@@ -0,0 +1,78 @@
...
+. In Exchange, click *Login* and supply your Anypoint Platform username and password.
+. Search for the Amazon S3 connector, click on the connector, and click *Add to Project*.
+. Follow the prompts.
+
```
Comment (george-hoffer) [inline modules/ROOT/pages/amazon/amazon-s3-connector-studio.adoc:21]: Provide a note for what operations can be input sources for this connector.

File: modules/ROOT/pages/amazon/amazon-s3-connector-studio.adoc [hunk 16]
```diff
@@ -0,0 +1,78 @@
...
+. Select *Amazon S3 Configuration (Configuration)*, and click *Edit*.
+. Specify the global element properties:
++
+image::amazon/amazon-s3-01.png[Global Element Properties]
++
+[%header%autowidth.spread]
+|===
+|Parameter |Description
+|Name |Name used to reference the connector instance.
+|Access Key |Alphanumeric text string that uniquely identifies the user who owns the AWS account.
```
Comment (george-hoffer) [inline modules/ROOT/pages/amazon/amazon-s3-connector-studio.adoc:39]: Need to say where to get these (they come from Amazon), but at least say that, and also check out Amazon to see how much work it is to find those values.

File: modules/ROOT/pages/amazon/amazon-s3-connector-studio.adoc [hunk 17]
```diff
@@ -0,0 +1,78 @@
...
++
+Access to operations on Amazon S3 is further controlled through policies. It is not always possible to validate your credentials before the exact operation for which you have access completes. This can vary based on the bucket name and other parameters. For example, the test connection can fail if your credentials have a restricted policy.
+
+. Click *OK* to save the global connector configurations.
+
+== Example of configuring S3-compatible Storage
+
+The https://github.com/minio/[Minio Project] is one example of Amazon S3 storage. If you configure this storage locally you might need to set the S3 Compatible Storage URL to `+http://127.0.0.1:9000+`.
+
+== Configure an Input Source
```
Comment (george-hoffer) [inline modules/ROOT/pages/amazon/amazon-s3-connector-studio.adoc:63]: This section should be near the top of this doc.

File: modules/ROOT/pages/amazon/amazon-s3-connector-xml-maven.adoc [hunk 18]
```diff
@@ -0,0 +1,72 @@
...
+        http/current/mule-http.xsd
+        http://www.mulesoft.org/schema/mule/s3
+        http://www.mulesoft.org/schema/mule/s3/current/mule-s3.xsd
+        http://www.mulesoft.org/schema/mule/ee/core
+        http://www.mulesoft.org/schema/mule/ee/core/current/mule-ee.xsd">
+----
+
+== Add a POM File Dependency
+
+The `pom.xml` Maven file generated by Anypoint Studio contains dependencies for a Mule app. When you code a Mule app manually, include this XML snippet in your `pom.xml` file to enable access to this connector. The Mule runtime engine translates the `LATEST` dependency keyword to the current version.
```
Comment (george-hoffer) [inline modules/ROOT/pages/amazon/amazon-s3-connector-xml-maven.adoc]: Found from the devs that LATEST doesn't exist. Use RELEASE instead.

File: modules/ROOT/pages/amazon/amazon-s3-connector-xml-maven.adoc [hunk 19]
```diff
@@ -0,0 +1,72 @@
...
+
+[source,xml,linenums]
+----
+<dependency>
+  <groupId>com.mulesoft.connectors</groupId>
+  <artifactId>mule-amazon-s3-connector</artifactId>
+  <version>LATEST</version>
```
Comment (george-hoffer) [inline modules/ROOT/pages/amazon/amazon-s3-connector-xml-maven.adoc]: Use RELEASE

**Review by george-hoffer — APPROVED** [review]

File: modules/ROOT/pages/amazon/amazon-s3-connector-studio.adoc (hunk 16 above)
Comment (dhirsch1) [reply modules/ROOT/pages/amazon/amazon-s3-connector-studio.adoc:39]: AWS is Amazon Web Services - I don't see the need to further qualify that this is AWS is an Amazon offering.
